lib/ansible/modules/identity/ipa/ipa_subca.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2017, Abhijeet Kasurde (akasurde@redhat.com)
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.0',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+
+DOCUMENTATION = '''
+---
+module: ipa_subca
+author: Abhijeet Kasurde (@akasurde)
+short_description: Manage FreeIPA Lightweight Sub CAs
+description:
+- Add, modify, enable, disable and delete an IPA Lightweight Sub CAs using IPA API
+version_added: "2.4"
+options:
+  subca_name:
+    description:
+    - The Sub Certificate Authority name which needs to be managed.
+    required: true
+  subca_subject:
+    description:
+    - The Sub Certificate Authority's Subject. For example., 'CN=SampleSubCA1,O=testrelm.test'
```

Comment:
Nit: You have an extra period after 'example'

Reply:
ok.